<launch>
	<node pkg="bill_diff_drive" type="diff_drive.py" name="cmd_vel_to_diff_drive" output="screen">
	</node>
</launch>
